### PR TITLE
pd: support pd real cluster

### PR DESCRIPTION
**Pull request by HuSharp**
rely on https://github.com/tikv/pd/pull/7493

### CODE REVIEW

File: pipelines/tikv/pd/latest/pod-pull_integration_realcluster_test.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        limits:
+          memory: 256Mi
+          cpu: 100m
+    - name: status-updater
+      image: hub.pingcap.net/pingcap/github-status-updater:latest
+      tty: true
+      resources:
+        limits:
+          memory: 256Mi
+          cpu: 100m
```
Comment (wuhuizuo) [inline pipelines/tikv/pd/latest/pod-pull_integration_realcluster_test.yaml:37]: not used
```suggestion
```

File: jobs/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        daysToKeep(30)
+    }
+    parameters {
+        // Ref: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables
+        stringParam("BUILD_ID")
+        stringParam("PROW_JOB_ID")
+        stringParam("JOB_SPEC", "", "Prow job spec struct data")
+    }
+    properties {
+        buildFailureAnalyzer(false) // disable failure analyze
```
Comment (wuhuizuo) [inline jobs/tikv/pd/latest/pull_integration_realcluster_test.groovy:14]:
```suggestion
        buildFailureAnalyzer(false)
```

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 3]
```diff
@@ -0,0 +1,67 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-pd"
+final GIT_CREDENTIALS_ID = 'github-sre-bot-ssh'
```
Comment (wuhuizuo) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:6]:
```suggestion
```

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 4]
```diff
@@ -0,0 +1,66 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-pd"
+final GIT_FULL_REPO_NAME = 'tikv/pd'
```
Comment (wuhuizuo) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:6]:
```suggestion
```

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        stage('Tests') {
+            options { timeout(time: 20, unit: 'MINUTES') }
+            steps {
+                dir('pd/') {
```
Comment (purelind) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:56]:
```suggestion
                dir('pd') {
```

File: pipelines/tikv/pd/latest/pod-pull_integration_realcluster_test.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.21:latest"
+      tty: true
+      resources:
+        requests:
+          memory: 12Gi
+          cpu: "4"
```
Comment (purelind) [inline pipelines/tikv/pd/latest/pod-pull_integration_realcluster_test.yaml:13]: It is recommended to set the values of "request" and "limit" to be the same.
```suggestion
          memory: 16Gi
          cpu: "6"
```

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**